server/src/test/java/io/spine/server/procman/ProcessManagerRepositoryTest.java

```diff
@@ -738,7 +815,8 @@ void deleteEntityViaMigration() {
 
         Optional<TestProcessManager> found = repository().find(id);
         assertThat(found.isPresent()).isTrue();
```

Comment:
There's a whole set of `Truth8` assertions that should allow having a bit nicer syntax for `Optional`s. Could you please check it out and consider using them instead?